tests/test_rat_command.py

```diff
@@ -35,16 +32,16 @@ def Setup_fx(bot_fx):
 @pytest.mark.commands
 @pytest.mark.usefixtures("Setup_fx")
 class TestRatCommand(object):
-
     @pytest.mark.asyncio
     async def test_invalid_command(self):
         """
-        Ensures the proper exception is raised when a command is not found.
-        :return:
+        Ensures that nothing happens and `trigger` exits quietly when no command can be found.
         """
-        with pytest.raises(CommandNotFoundException):
+        try:
             await Commands.trigger(message="!nope", sender="unit_test",
                                    channel="foo")
+        except BaseException as e:
+            pytest.fail("trigger raised " + type(e).__name__)
```

Comment:
why catch the exception, methinks pytest will give better introspection if we let it catch the exception instead.

unhandled / unexpected exceptions in tests automatically fail a test.

Reply:
You're right, I think I changes this one about a couple of times, but in the end I went for the explicit fail to make it clear that `trigger` not raising an exception is what was being tested.

`with pytest.does_not_raise:` is something I would have preferred but it doesn't exist.